demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1215,12 +1216,19 @@ def validate_deleted_files(self, deleted_files, added_files) -> bool:
 
         is_valid = True
         for file_path in deleted_files:
-            file_path = str(file_path)
-            if not self.was_file_renamed_but_labeled_as_deleted(file_path, added_files):
-                if not self.is_file_allowed_to_be_deleted(file_path):
-                    error_message, error_code = Errors.file_cannot_be_deleted(file_path)
-                    if self.handle_error(error_message, error_code, file_path):
-                        is_valid = False
+            if 'Packs' not in Path(file_path).absolute().parts:
+                # not allowed to delete non-content files
+                error_message, error_code = Errors.file_cannot_be_deleted(file_path)
```

Comment:
file_path is passed here, is it a string or a `Path` object? Please add type hints to the method signature if possible.

Further down (row 1226) we call `str()` on it, but not here, why?

Reply:
It is both. When using the -g flag we get a Path object. When using another flag we get a str

Reply:
Oh boy. Please add type hints as `List[Union[str,Path]]` to this method, add `file_path = Path(file_path)` to the beginning of the method, then later call `str` before sending to `Errors.some_error(path)`.

Reply:
Actually it is a Set, the precommit check fails when adding typing of `Set[Union[Path,str]]`, instead I added just the `set`. Let me know if you have a better solution